spec/functional/account.spec.js

```diff
@@ -46,8 +46,8 @@ describe("account", () => {
             });
         }).then((response) => {
             expect(response.statusCode).toBe(200);
-            expect(response.headers["Content-Type"]).toBe("text/plain");
-            expect(response.body.toString("binary")).toBe("This is a test token");
+            expect(response.getHeader("Content-Type")).toBe("text/plain");
+            expect(response.body.toString()).toBe("This is a test token\n");
```

Comment:
Looks like the newline is added in this one by lib/formatter/text-formatter.js on this line

```
return done(null, new Buffer(`${content}\n`, "binary"));
```